### PR TITLE
[konflux] bump pipeline run template

### PR DESCRIPTION
**Pull request by ashwindasr**
Right now, we are using an old template. 

Most of the params are overriden, but we need to double check, but this is the new template from https://github.com/openshift-priv/art-konflux-template/blob/main/.tekton/art-konflux-template-pulkl-request.yaml

The next step is to pull directly from that repo, instead of storing it in art-tools

### CODE REVIEW

**Comment by ashwindasr** [comment]
/hold

Until we do a test build

**Comment by ashwindasr** [comment]
/unhold

Test build [succeeded](https://konflux.apps.stone-prod-p02.hjvn.p1.openshiftapps.com/application-pipeline/workspaces/ocp-art/applications/openshift-4-18/pipelineruns/doozer-build-openshift-4-18-openshift-base-rhel9-9x5s4)

File: doozer/doozerlib/backend/konflux_image_build_pipelinerun.yaml [hunk 1]
```diff
@@ -167,40 +185,38 @@ spec:
...
+        workspace: workspace
       - name: git-basic-auth
         workspace: git-auth
       - name: netrc
         workspace: netrc
-    - matrix:
-        params:
-        - name: PLATFORM
-          value:
-          - $(params.build-platforms)
```
Comment (thegreyd) [inline doozer/doozerlib/backend/konflux_image_build_pipelinerun.yaml:202]: Looking at the build that got created, built for only 1 arch, https://konflux.apps.stone-prod-p02.hjvn.p1.openshiftapps.com/application-pipeline/workspaces/ocp-art/applications/openshift-4-18/pipelineruns/doozer-build-openshift-4-18-openshift-base-rhel9-9x5s4/logs

Comment (thegreyd) [reply doozer/doozerlib/backend/konflux_image_build_pipelinerun.yaml:202]: is this the multi arch template?

**Comment by ashwindasr** [comment]
/hold

Looks like we need to onboard a multi-arch component.

**Comment by ashwindasr** [comment]
since this config was the correct one. Copied over to openshift-priv: https://github.com/openshift-priv/art-konflux-template/pull/7

will wait a couple of days to see if konflux auto updates that file

**Pull request closed without merging**